src/components/PageHeader/PageHeader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PageHeader.scss';
+
+const PageHeader = ( { border, children, intro, title } ) => (
+	<div className={`${styles.container} ${border ? styles.border : ''}`}>
+		<div className={styles.contents}>
+			<h1 className={styles.title}>{title}</h1>
+			<p className={styles.intro}>{intro}</p>
+		</div>
+		{children}
+	</div>
+);
+
+PageHeader.propTypes = {
```

Comment:
describe each prop with a docblock (see example component)

Reply:
Isn't this what `ArgsTable` is for? I'm not sure what I should be looking for in `Example`

Reply:
`ArgsTable` and `PropsTable` pull the `description` of each prop from a comment block above the proptype definition. See https://github.com/Quartz/interface/blob/main/src/components/Example/Example.jsx#L12

Reply:
Derp! thank you